docs/Building-Transmission.md

```diff
@@ -120,24 +120,34 @@ $ sudo make install
 ```
 
 ## On Windows ##
-For Windows XP and above there are several choices:
 
-### Cygwin environment ###
-With Cygwin https://cygwin.com/ installed, the CLI tools (transmission-remote, transmission-cli, etc.) and the daemon can be built easily.
+## Building transmission-daemon
+You need the following installed:
 
-No patches needed(\*), all the recent versions of Transmission build almost out-of-the-box (you need to install the prerequisites), and the CLI tools work better under Cygwin than those built with MinGW.
+* Visual Studio 2019 or greater (the Community Edition is sufficient - just make sure its C++ compiler, MSVC, is installed)
+* [ActivePerl](https://www.activestate.com/products/activeperl/) or [StrawberryPerl](https://strawberryperl.com)
+* [CMake](https://cmake.org/download/) (choose to add CMake to your path)
+* Possibly [Git for Windows](https://git-scm.com/download/win)
+* [Vcpkg](https://github.com/microsoft/vcpkg#quick-start-windows)
 
-(\*) At the release time of version 2.0, **libevent** is not bundled and it is also not in the Cygwin distribution (but was added later)... so you need to build it (which is as easy as ./configure, make install). To build Transmission you may need to add LDFLAGS="-L/usr/local/lib" to the configure script (LIBEVENT_LIBS does not seem to work when it comes to build all the test programs).  Additionally **libutp** needs deleting -ansi on the Makefile.
 
-With version 2.51 miniupnpc fails to build, see https://miniupnp.tuxfamily.org/forum/viewtopic.php?t#1130.
+### Install all dependencies through vcpkg
 
-Version 2.80 breaks building on Cygwin, adding this https://github.com/adaptivecomputing/torque/blob/master/src/resmom/cygwin/quota.h file to Cygwin's /usr/include/sys solves the problem.  This is no longer needed after version 2.82 (Cygwin added the header).
-
-Version 2.81 with the above workaround needs a one line patch, see ticket #5692.
-
-Version 2.82, same as 2.81.
+```
+vcpkg integrate install
+vcpkg install curl
+vcpkg install zlib
+vcpkg install openssl
+```
 
-Version 2.83, no need to add quota.h, Cygwin added it.
+### Get Transmission source and build it
+```
+git clone https://github.com/transmission/transmission
+cd transmission
+cmake -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_TOOLCHAIN_FILE=*vcpkg root*/scripts/buildsystems/vcpkg.cmake
```

Comment:
`*vcpkg root*` here should either be some sort of variable, e.g. `$($VCPKG_ROOT)` in powershell, or it should be an actual path.

Reply:
Also, this is not a correct path on windows unless you're using bash

Reply:
The path (if you mean forward slashes) is perfectly fine for both CMake and Windows.

Reply:
I was under the mistaken impression that powershell enforced backslash paths for compatibility, but that appears to be incorrect. CMake, of course, will take forward slashes but that is about the shell.

Reply:
But 

> `*vcpkg root*` here should either be some sort of variable, e.g. `$($VCPKG_ROOT)` in powershell, or it should be an actual path.

But at first we need to set it?

Reply:
The point is to make it clear that it's a variable. In powershell, the most likely shell a windows user would use to build transmission, a variable substituted into a string looks like `"$($VCPKG_ROOT)"`.

Reply:
Powershell... Hm.. Maybe %VCPKG_ROOT% for usual cmd?